src/main/java/com/minecolonies/util/BlockUtils.java

```diff
@@ -106,4 +107,132 @@ public static boolean isWater(IBlockState iBlockState)
+        }
+        else if(block instanceof BlockCauldron)
+        {
+            return Items.cauldron;
+        }
+        else if(block instanceof BlockCocoa)
+        {
+            return Items.dye;
+        }
+        else if(block instanceof BlockCrops)
+        {
+            if(block instanceof BlockCarrot)
+            {
+                return Items.carrot;
+            }
+            else if(block instanceof BlockPotato)
+            {
+                return Items.potato;
+            }
+            return Items.wheat_seeds;
+        }
+        else if(block instanceof BlockDaylightDetector)
+        {
+            return Item.getItemFromBlock(Blocks.daylight_detector);
+        }
+        else if(block instanceof BlockDoor)
+        {
+            return block == Blocks.iron_door ? Items.iron_door : (block == Blocks.spruce_door ? Items.spruce_door :
+                                                                  (block == Blocks.birch_door ? Items.birch_door :
+                                                                   (block == Blocks.jungle_door ? Items.jungle_door :
+                                                                    (block == Blocks.acacia_door ? Items.acacia_door :
+                                                                     (block == Blocks.dark_oak_door ? Items.dark_oak_door : Items.oak_door)))));
+        }
+        else if(block instanceof BlockFarmland)
+        {
+            return Item.getItemFromBlock(Blocks.dirt);
+        }
+        else if(block instanceof BlockFlowerPot)
+        {
+            return Items.flower_pot;
+        }
+        else if(block instanceof BlockFurnace)
+        {
+            return Item.getItemFromBlock(Blocks.furnace);
+        }
+        else if(block instanceof BlockNetherWart)
+        {
+            return Items.nether_wart;
+        }
+        else if(block instanceof BlockRedstoneComparator)
+        {
+            return Items.comparator;
+        }
+        else if(block instanceof BlockRedstoneLight)
+        {
+            return Item.getItemFromBlock(Blocks.redstone_lamp);
+        }
+        else if(block instanceof BlockRedstoneRepeater)
+        {
+            return Items.repeater;
+        }
+        else if(block instanceof BlockRedstoneTorch)
+        {
+            return Item.getItemFromBlock(Blocks.redstone_torch);
+        }
+        else if(block instanceof BlockRedstoneWire)
+        {
+            return Items.redstone;
+        }
+        else if(block instanceof BlockReed)
+        {
+            return Items.reeds;
+        }
+        else if(block instanceof BlockSign)
+        {
+            return Items.sign;
+        }
+        else if(block instanceof BlockSkull)
+        {
+            return Items.skull;
+        }
+        else if(block instanceof BlockStem)
+        {
+            Item item =  block == Blocks.pumpkin ? Items.pumpkin_seeds : (block == Blocks.melon_block ? Items.melon_seeds : null);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'item' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)